examples/apps/src/ble_bas_peripheral.rs

```diff
@@ -10,7 +10,7 @@ const CONNECTIONS_MAX: usize = 1;
 const L2CAP_CHANNELS_MAX: usize = 2; // Signal + att
 
 // GATT Server definition
-#[gatt_server]
+#[gatt_server(connections_max = CONNECTIONS_MAX)]
```

Comment:
I think we should have a default of '1' in the macro, which covers most of the common 'personal accessory' use cases